packages/xo-server-load-balancer/src/plan.js

```diff
@@ -314,9 +314,23 @@ export default class Plan {
   async _processVcpuPrepositioning(hosts) {
     const promises = []
 
-    const idToHost = keyBy(hosts, 'id')
+    // removing hosts which have incorrect cpu count value to avoid mass migration on rrd malfunction
+    const sanitizedHostList = hosts.filter(host => parseInt(host.cpus.cores) > 0)
```

Comment:
cpus.cores is `number | undefined`. You don't need `parseInt`
```suggestion
    const sanitizedHostList = hosts.filter(host => host.cpus.cores > 0)
```